assets/src/edit-story/app/font/actions/useLoadFontFiles.js

```diff
@@ -25,38 +25,86 @@ import { useCallback } from 'react';
 import cleanForSlug from '../../../utils/cleanForSlug';
 import getGoogleFontURL from '../../../utils/getGoogleFontURL';
 
+/**
+ * This is a utility ensure that Promise.all return ONLY when all promises are processed.
```

Comment:
To clarify, this method ensures that the individual failures are tolerated, right?

Reply:
Yes. This avoid canceling the whole promise because of a single failure (useful mainly for MULTIPLE special value).

Reply:
This is ok. And you can also do `promise.catch(() => {/* do nothing. the promise will no longer fail*/})`.